src/webgpu/api/validation/attachment_compatibility.spec.ts

```diff
@@ -0,0 +1,305 @@
+export const description = `
+Validation for attachment compatibility between render passes, bundles, and pipelines
+
+TODO: Add sparse color attachment compatibility test when defined by specification
+`;
+
+import { poptions, params } from '../../../common/framework/params_builder.js';
+import { makeTestGroup } from '../../../common/framework/test_group.js';
+import { range } from '../../../common/framework/util/util.js';
+import {
+  kRegularTextureFormatInfo,
+  kRegularTextureFormats,
+  kSizedDepthStencilFormats,
+  kUnsizedDepthStencilFormats,
+  RegularTextureFormat,
+} from '../../capability_info.js';
+
+import { ValidationTest, CommandBufferMaker } from './validation_test.js';
+
+// TODO: Update maximum color attachments when defined
+const kColorAttachmentCounts = range(4, i => i + 1);
```

Comment:
I guess this should be in capability_info too

Reply:
Done